tests/src/runtest.proj

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="src\dir.props" />
+  <Import Project="Directory.Build.props" />
```

Comment:
This also should probably be an sdk style project.

Reply:
Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.